package.json

```diff
@@ -2,17 +2,28 @@
   "name": "react-directus",
   "version": "0.0.2",
   "description": "A set of React components and utilities for Directus Headless CMS",
+  "homepage": "https://github.com/gremo/react-directus",
```

Comment:
Does homepage and repository url used in the npm registry? https://www.npmjs.com/package/react-directus

Reply:
Yes, on the right side the `Repository` and `Homepage` are displayed.
See [Example](https://www.npmjs.com/package/@directus/sdk)

Reply:
If #360 is merged ad Github pages is enabled we would even have a homepage to link to.